package.json

```diff
@@ -23,7 +23,7 @@
     "start:docs": "DOCS_ENV=dev parcel 'packages/@react-{spectrum,aria,stately}/*/docs/*.mdx' 'packages/react-aria-components/docs/**/*.mdx' 'packages/@internationalized/*/docs/*.mdx' 'packages/dev/docs/pages/**/*.mdx'",
     "build:docs": "DOCS_ENV=staging parcel build 'packages/@react-{spectrum,aria,stately}/*/docs/*.mdx' 'packages/react-aria-components/docs/**/*.mdx' 'packages/@internationalized/*/docs/*.mdx' 'packages/dev/docs/pages/**/*.mdx'",
     "test": "cross-env STRICT_MODE=1 yarn jest",
-    "test-loose": "yarn jest",
+    "test-loose": "cross-env VIRT_ON=1 yarn jest",
```

Comment:
virtualizer is off if the user doesn't mock the window's clientWidth/clientHeight, but it is always on for our tests. For people's current tests they probably already have these mocks since virtualizer doesn't really work without those so things should theoretically run the same in test. The `VIRT_ON` here is just to make sure virtualizer is on for our tests always

Reply:
ah, right, ok, i follow the logic now. thanks